docs/Upgrade-Guide/upgrade_to_6.md

```diff
@@ -0,0 +1,81 @@
+## Upgrade procedure to Gluster 6, from Gluster 5.x, 4.1.x, and 3.12.x
```

Comment:
Don't we need to add this file to the mkdocs.yml file at the root of the repo? I always thought the build depended on the same.

Reply:
Done